MSAL/test/integration/native_auth/end_to_end/MSALNativeAuthEndToEndBaseTestCase.swift

```diff
@@ -129,4 +172,10 @@ class MSALNativeAuthEndToEndBaseTestCase: XCTestCase {
             return Constants.clientIdEmailCodeAttributesKey
         }
     }
+    
+    private func getCustomTenantSubdomain(tenantName: String?, tenantId: String?, format: Int) -> String {
+        return Constants.customDomainFormat[format]
+            .replacingOccurrences(of: "<tenantName>", with: tenantName!)
```

Comment:
Misspelling the string to replace can led to unexpected result. Did you consider using String template instead? 
More about it here:
https://developer.apple.com/documentation/swift/string/init(format:_:)
Moreover, `tenantSubdomain` and `tenantId` should not be nil at this point. 
You can use String template like this:
`let stringresult = String(format: "https://%@.ciamlogin.com/", tenantSubdomain)`

Reply:
Good idea. Done.